server/dataone_register.py

```diff
@@ -175,12 +211,22 @@ def find_initial_pid(path):
         return path
 
 
-def get_aggregated_identifiers(pid):
-    """Process an OAI-ORE aggregation into a set of aggregated identifiers."""
+def get_aggregated_identifiers(pid, base_url):
```

Comment:
if this isn't being used anymore and we use version control, could this get removed entirely?

Reply:
I'll go ahead and take the unused code out